news/flake8_in_CI.rst

```diff
@@ -0,0 +1,26 @@
+**Added:**
+
+* CI step to run flake8 after pytest.
+
+**Changed:**
+
+* Updated pytest_plugin for pytest 5.4 API, pip requirements for pytest>= 5.4
+
+**Deprecated:**
+
+* `pytest --flake8`_ now exits with error message to use flake8 instead.
```

Comment:
I think, the trailing underscore after ``` `pytest --flake8` ``` shouldn't be here, right? Also, I think, you wanted to write ``` ``pytest --flake8`` ``` (with double backticks).

Reply:
I was going for a link highlight, which this does.  But I see that's not how it's done in other news files....

Reply:
Thanks!